user-registration-app/src/app/views/registration-form/registration-form.component.ts

```diff
@@ -161,21 +161,28 @@ export class RegistrationFormComponent implements OnInit {
 
   ngOnInit() {
     this.route.data.subscribe(({ registration }) => {
-      this.registrationForm = registration;
-      this.registrationForm.firstHackathon = false;
-      this.registrationForm.eighteenBeforeEvent = null;
-      this.registrationForm.mlhcoc = null;
-      this.registrationForm.mlhdcp = null;
-      this.parsePhone(this.registrationForm.phone);
-      this.phoneNoUse = this.prettifiedPhone;
-      this.diet_restr = this.registrationForm.dietaryRestriction !== null;
+      this.registrationForm = new Registration();
+      //Data to keep
+      this.nukeData(this.registrationForm, registration)
```

Comment:
Is there a better name that could be used for this function?

Reply:
Probably, but this is already in production. If you look at "files changed" instead of the commits it shouldn't be there? or at least it isnt for me

Reply:
Does the outdated label mean the file has been updated already?

Reply:
yep. It should be updated because of the merge master into dev commit

Reply:
Lol I'm dumb